docker/deployer/Dockerfile

```diff
@@ -90,19 +80,16 @@ COPY --from=builder /app/kakarot/tests ./tests/
 
 # Cairo Smart contracts are used in deploy_kakarot.py
 # To limit the probability of this Dockerfile to break, we copy the entire src and not individual files
-COPY --from=builder /app/kakarot/src ./src/
+COPY --from=builder /app/kakarot/kakarot0/src ./src/
 
 # Default Solidity contracts are also used in deploy_kakarot.py
 COPY --from=builder /app/kakarot/solidity_contracts ./solidity_contracts/
 
 COPY --from=builder /app/kakarot/Makefile .
-COPY --from=builder /app/kakarot/poetry.lock .
+COPY --from=builder /app/kakarot/uv.lock .
 COPY --from=builder /app/kakarot/pyproject.toml .
 COPY --from=builder /app/kakarot/foundry.toml .
 COPY --from=builder /app/kakarot/README.md .
 
-RUN --mount=type=cache,target=/root/.cache \
-    poetry install --without dev
-
 # Deploy kakarot
-CMD ["python", "kakarot_scripts/deploy_kakarot.py"]
+CMD ["uv", "run", "python", "kakarot_scripts/deploy_kakarot.py"]
```

Comment:
```suggestion
CMD ["uv", "run", "kakarot_scripts/deploy_kakarot.py"]
```

Reply:
see added project scripts

```
[project.scripts]
compile = "kakarot_scripts.compile_kakarot:main"
deploy = "kakarot_scripts.deploy_kakarot:main_sync"
ef_tests = "kakarot_scripts.ef_tests.fetch:generate_tests"
```

so `uv run compile` will compile and `uv run deploy` will deploy